intercom-java/src/main/java/io/intercom/api/Event.java

```diff
@@ -50,6 +53,23 @@ public static Job submit(List<JobItem<Event>> items, Job job)
         return Job.listJobErrorFeed(jobID, Event.class);
     }
 
+    public static EventCollection listBy(String type, String parameterName, String parameterValue) {
```

Comment:
I think to keep things consistent with the rest of the library and our Idioms https://github.com/intercom/intercom-java/#pagination, this should be refactored to be a `list` call which takes an input of `Map<String, String> params`. Seems like we could utilise a similar pattern as the Notes and NotesCollection here 😄 

Some examples of using `list` in the current code
- https://github.com/intercom/intercom-java/blob/f50c869194eac5d4fc047e08634b99c14334e88b/intercom-java/src/main/java/io/intercom/api/Note.java#L26
- https://github.com/intercom/intercom-java/blob/f3cf4b23cb49fded5fc6e2214ad0d95bd7ee7676/intercom-java/src/main/java/io/intercom/api/Company.java#L56

It would be good to also add some sample code to the [README.md](https://github.com/intercom/intercom-java/blob/master/README.md) file to indicate how to utilise this new endpoint